diff --git a/config.h b/config.h
new file mode 100644
index 0000000..4964ed4
--- /dev/null
+++ b/config.h
@@ -0,0 +1,37 @@
+/* config.h.in.  Generated automatically from configure.in by autoheader.  */
+
+/* Define if you need to in order for stat and other things to work.  */
+#undef _POSIX_SOURCE
+
+/* Define if you have the ANSI C header files.  */
+#define STDC_HEADERS
+
+/* Define if the X Window System is missing or not being used.  */
+#undef X_DISPLAY_MISSING
+
+/* Standard C headers present */
+#undef STDC_HEADERS
+
+/* Build for OpenGL by default, and not for old IrisGL aka GL 3.2 */
+#define  OPENGL_10 1          
+#undef   GL_32 
+
+/* Disable debugging stuff (debugging replaces GL output with printfs) */
+#undef   DEBUG_OUTPUT
+
+/* Do we have a lenient tesselator? */
+#undef  LENIENT_TESSELATOR
+#define DELICATE_TESSELATOR 1
+
+/* Enable texture mapping by default. */
+#define AUTO_TEXTURE 1
+
+/* Define if you have the memcpy function.  */
+#define HAVE_MEMCPY
+
+/* Name of package */
+#undef PACKAGE
+
+/* Version number of package */
+#undef VERSION
+
diff --git a/gle.def b/gle.def
new file mode 100644
index 0000000..49a2d9b
--- /dev/null
+++ b/gle.def
@@ -0,0 +1,33 @@
+;
+
+LIBRARY		GLE32
+DESCRIPTION	"GLE OpenGL Extrusion library"
+CODE		PRELOAD MOVEABLE DISCARDABLE
+DATA		PRELOAD MOVEABLE MULTIPLE
+
+EXPORTS
+	gleExtrusion
+	gleGetJoinStyle
+	gleGetNumSides
+	gleHelicoid
+	gleLathe
+	glePolyCone
+	glePolyCylinder
+	gleScrew
+	gleSetJoinStyle
+	gleSetNumSides
+	gleSpiral
+	gleSuperExtrusion
+	gleTextureMode
+	gleToroid
+	gleTwistExtrusion
+	rot_about_axis
+	rot_axis
+	rot_omega
+	rot_prince
+	urot_about_axis
+	urot_axis
+	urot_omega
+	urot_prince
+	uview_direction
+	uviewpoint
\ No newline at end of file
